spec/data/normal.yml

```diff
@@ -258,6 +258,10 @@ paths:
                     mapping:
                       obj1: '#/components/schemas/one_of_object1'
                       obj2: '#/components/schemas/one_of_object2'
+                one_of_with_nullable:
+                  oneOf:
+                    - nullable: true
```

Comment:
First, when we want to enable nullable, you need to set `type` field

```
A true value adds "null" to the allowed type specified by the type keyword, only if type is explicitly defined within the same Schema Object.
```
https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#fixed-fields-20

But this gem doesn't check `type` keyword, you get error.
I think we should ignore nullable when `type` keyword doesn't exist.

(And when you set `type: object`, this schema match all object and validation failed because match both schema in oneOf so you need set `type: boolean` or other types)

Reply:
@ota42y Thank you for the quick response!

Sorry I didn't read the document carefully. Actually I was confused how to define the schema correctly. I wanted to let the schema accept either `nil` or `one_of_object1`. At the same time, since I use `one_of_object1` at other places and don't want it be nullable, I used `oneOf` and added `nullable: true` at its same level. It worked perfect until the `unspecified_type` was introduced.

Then I think the correct way is to define another schema for `nullable_one_of_object1`.

Reply:
This is such a confusing area that we had to change the wording to be more precise in 3.0.3, so don't worry about it!